src/styles/font.ts

```diff
@@ -0,0 +1,120 @@
+import { Platform, TextStyle } from 'react-native'
+
+import { MixedSizeCSSPropertiesKeys } from 'react-native-render-html'
+
+import theme from './theme'
+
+/**
+ * Reacht Native's TextStyle without the overflow:'scroll' property.
+ *
+ * The component @native-html (used by react-native-render-html) is missing the overflow: 'scroll' property.
+ * In order to use the React Native's TextStyle together with the RenderHtml component (and type checking)
+ * we override the overflow property with our own overflow type
+ */
+type FontStyle = TextStyle & { overflow?: 'visible' | 'hidden' | undefined } & {
+  [k in MixedSizeCSSPropertiesKeys]?: number | string
```

Comment:
Hier voeg ik `[k in MixedSizeCSSPropertiesKeys]?: number | string` toe zodat de typing weer klopt voor h1, h2 en h3 typing in `src/styles/html.ts`. Die was veranderd door `DimensionValue`.